lib/Connection.js

```diff
@@ -79,8 +79,15 @@ class Connection {
     if (!this.socket) {
       callback(new Error('No open SIP2 socket connection'));
     }
-    this.socket.once('data', data => callback(null, parseResponse(data)));
-    this.socket.once('error', err => callback(err));
+	let data='';
+	this.socket.on('data', function getData (packet) {
+		data+= packet;
+		if( /(\r\n|\r|\n)$/.exec(packet) !== null){
+        	this.removeListener('data', getData);
+			callback(null, parseResponse(data));
+		}
+    });
```

Comment:
```suggestion
	let data='';
	this.socket.on('data', function getData (packet) {
		data+= packet;
		if( /(\r\n|\r|\n)$/.exec(packet) !== null){
        	        this.removeListener('data', getData);
			callback(null, parseResponse(data));
		}
        });
```

looks like the indentation might be slight off.